backend/main.py

```diff
@@ -192,6 +191,9 @@ async def run_startup_tasks():
 app.include_router(agent_templates.router, prefix="/api")
 
 if os.getenv("DISABLE_ATTACHMENTS") != "true":
```

Comment:
Since we have multiple places check for `os.getenv("DISABLE_ATTACHMENTS") != "true"`, maybe we should create a centralize a helper in backend/utils/auth_utils.py.
```
def is_attachments_enabled() -> bool:
      """
      Check if attachments feature is enabled.
      
      Returns:
          bool: True if attachments should be enabled, False otherwise
      """
      return os.getenv("DISABLE_ATTACHMENTS") != "true"
```
And then just do:
`from ..utils.auth_utils import is_attachments_enabled`

`if is_attachments_enabled():`

Reply:
I have incorporated this change.